### PR TITLE
Added DashLineCap, StartLineCap, EndLineCap to the Shape, fixed bug.

### LINKED ISSUE #313: Direct2D LineCap is not set for Pen
Direct2D LineCap is not set for Pen
caps_d2d

Skia version with proper LineCap's
caps_skia

### PR DESCRIPTION
**Pull request by susloparovdenis**
Fixes #313

### CODE REVIEW

File: src/Perspex.Controls/Shapes/Shape.cs [hunk 1]
```diff
@@ -90,13 +90,20 @@ public double StrokeThickness
             set { SetValue(StrokeThicknessProperty, value); }
         }
 
+        public PenLineCap DashCap { get; set; } = PenLineCap.Flat;
```
Comment (grokys) [inline src/Perspex.Controls/Shapes/Shape.cs]: These properties in WPF are name `StrokeDashCap`, `StrokeStartLineCap` etc - we should probably use the same naming convention here.

**Comment by grokys** [comment]
Also, the Cairo test is failing, I assume because you've not implemented it for Cairo. Could you disable the test when running under Cairo if this is the case? See here for an example of how to do that:

https://github.com/Perspex/Perspex/blob/master/tests/Perspex.RenderTests/Media/VisualBrushTests.cs#L88

**Comment by susloparovdenis** [comment]
https://github.com/Perspex/Perspex/issues/313#issuecomment-162973787
Is it OK? :)

**Comment by grokys** [comment]
Looks good to me!

**Pull request merged**